sdks/python/apache_beam/ml/inference/tensorrt_inference.py

```diff
@@ -19,7 +19,6 @@
 
 import numpy as np
 import os
-import pycuda.autoinit
```

Comment:
@yeandy I would advise against deleting this line. [Autoinit](https://documen.tician.de/pycuda/util.html) automatically performs all the steps necessary to get CUDA ready to execute compute kernels. You would get this if you take it out:
![image](https://user-images.githubusercontent.com/101208641/178040411-040e82d9-b148-4891-87eb-2d93b8c2bc24.png)

Reply:
Sounds good. I was getting a `pylint` error, but we can configure it to be ignored.